install/k8s/contiv/contiv.yaml

```diff
@@ -90,20 +143,63 @@ spec:
     metadata:
       labels:
         k8s-app: contiv-netplugin
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       hostNetwork: true
       hostPID: true
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      serviceAccountName: contiv-netplugin
+      initContainers:
+        - name: contiv-netplugin-init
+          image: contiv/netplugin-etcd-init:latest
+          imagePullPolicy: Always
```

Comment:
Could this result in an incompatible init container being pulled down and run with an older release of Contiv?

Reply:
it could be, but we have to do it before we starting tagging `netplugin-etcd-init`